### ISSUE TITLE
[BUG] Non-text elements lack sufficient colour contrast.

### ISSUE THREAD

**Issue by DaveTryon** [microsoft/accessibility-insights-windows#1758]
(This is split out from #1738)

From @delphinegilbert:

Hello,

An accessibility audit was performed on Accessibility Insights for Windows in our department and the following issue was found:

**Description of issue**

Right under "Live Inspect", there is a search input field that lacks any kind of visible border which makes it difficult to see. The input field's colour contrast fails against the white background.

**Impact on users**

This will especially impact people with low vision and non-keyboard, as users may not see at all that they the input field is there.

**Steps to reproduce**

Launch the program, and dismiss the startup windows. Using your mouse, hover or click on any other window for the program to start inspecting. It should then display an item tree, there there's a search field at the top.

![image](https://github.com/microsoft/accessibility-insights-windows/assets/59966346/d77bf17b-27ff-4d09-b518-b9e4ea714601)

**Comment by DaveTryon** [comment]
This is live in Canary and will be included in our next Production release
